providers/flagd/README.md

```diff
@@ -42,19 +42,40 @@ openfeature.SetProvider(provider)
 
 | Option name                                              | Environment variable name      | Type & supported value      | Default   | Compatible resolver |
 |----------------------------------------------------------|--------------------------------|-----------------------------|-----------|---------------------|
-| WithHost                                                 | FLAGD_HOST                     | string                      | localhost | RPC & in-process    |
-| WithPort                                                 | FLAGD_PORT                     | number                      | 8013      | RPC & in-process    |
-| WithTLS                                                  | FLAGD_TLS                      | boolean                     | false     | RPC & in-process    |
-| WithSocketPath                                           | FLAGD_SOCKET_PATH              | string                      | ""        | RPC & in-process    |
-| WithCertificatePath                                      | FLAGD_SERVER_CERT_PATH         | string                      | ""        | RPC & in-process    |
-| WithLRUCache<br/>WithBasicInMemoryCache<br/>WithoutCache | FLAGD_CACHE                    | string (lru, mem, disabled) | lru       | RPC                 |
-| WithEventStreamConnectionMaxAttempts                     | FLAGD_MAX_EVENT_STREAM_RETRIES | int                         | 5         | RPC                 |
+| WithHost                                                 | FLAGD_HOST                     | string                      | localhost | rpc & in-process    |
+| WithPort                                                 | FLAGD_PORT                     | number                      | 8013      | rpc & in-process    |
+| WithTLS                                                  | FLAGD_TLS                      | boolean                     | false     | rpc & in-process    |
+| WithSocketPath                                           | FLAGD_SOCKET_PATH              | string                      | ""        | rpc & in-process    |
+| WithCertificatePath                                      | FLAGD_SERVER_CERT_PATH         | string                      | ""        | rpc & in-process    |
+| WithLRUCache<br/>WithBasicInMemoryCache<br/>WithoutCache | FLAGD_CACHE                    | string (lru, mem, disabled) | lru       | rpc                 |
+| WithEventStreamConnectionMaxAttempts                     | FLAGD_MAX_EVENT_STREAM_RETRIES | int                         | 5         | rpc                 |
+| WithEventStreamConnectionMaxAttempts                     | FLAGD_MAX_EVENT_STREAM_RETRIES | int                         | 5         | rpc                 |
+| WithOfflineFilePath                                      | FLAGD_OFFLINE_FLAG_SOURCE_PATH | string                      | ""        | in-process          |
```

Comment:
The Node.JS version doesn't support the offline environment variable. Is this supported in Java? Maybe we should create a ticket and include it in the [in-process provider spec](https://flagd.dev/reference/specifications/in-process-providers/#environment-variables).

Reply:
No, we don't have that for the Java provider. I will follow-up on this for the Java provider and create an issue for JS provider.